src/Aspire.Hosting/ApplicationModel/CommandsConfigurationExtensions.cs

```diff
@@ -44,7 +44,7 @@ internal static void AddLifeCycleCommands(this IResource resource)
                     return ResourceCommandState.Hidden;
                 }
             },
-            displayDescription: null,
+            displayDescription: "Start resource",
```

Comment:
Any ideas on how we could localize these? Options including having the server send all translated values, or sending a known key value that's then looked up in the dashboard.

Reply:
I don't think there is any plan for this kind of text to be localizable. Like state text, the API for adding commands just takes one string. That would need to change, and then I guess the server would need to send the frontend lang in a cookie or query string.